modules/installation-configuration-parameters.adoc

```diff
@@ -192,7 +192,12 @@ Only IPv4 addresses are supported.
 endif::agent,bare,ibm-power,ibm-z,vsphere[]
 
 ifdef::agent,bare,ibm-power,ibm-z,vsphere[]
+Consider the following information before you configure network parameters for your cluster:
```

Comment:
According to [Glenn's comment](https://issues.redhat.com/browse/OCPBUGS-34504#:~:text=Two%20solutions%20for%20this%3A) on this Jira, should I mention DHCP?

Reply:
We could mention that if you have v6 DHCP in your infrastructure and don't want to use v6 in your cluster you can disable v6 on the primary interface using NMState.

Reply:
Thanks. Added:

> If your infrastructure uses IPv6 as the primary network layer protocol in a Dynamic Host Configuration Protocol (DHCP) environment and your cluster does not support IPv6, you can use the Kubernetes NMState Operator to disable IPv6 on your infrastructure